annex/include/temperature_data.h

```diff
@@ -21,4 +20,3 @@ class TemperatureData {
         kpsr::Subscriber<int> * _subscriber;
         kpsr::Publisher<int> * _publisher;
 };
```

Comment:
This whole file should be deleted from the repo, not just the #ifndef line. Use `git rm` on the command line to remove this file.

Reply:
File removed